api/seeds/shared/roles-and-activities.js

```diff
@@ -135,4 +177,5 @@ exports.seed = async knex => {
       'eAPD State SME'
     ])
     .update({ isActive: false });
+  logger.info('Updated active status for roles');
```

Comment:
Running the seed command on my local database yields the appropriate associations in these tables.